setup.py

```diff
@@ -119,14 +116,18 @@ def get_exts_for(name):
         # enum34 is an enum backport for earlier versions of python
         # funcsigs backport required for vendored debtcollector
         # encoding using msgpack
-        install_requires=["enum34; python_version<'3.4'", "funcsigs>=1.0.0; python_version=='2.7'", "msgpack>=0.5.0"],
+        install_requires=[
+            "enum34; python_version<'3.4'",
+            "funcsigs>=1.0.0; python_version=='2.7'",
+            "msgpack>=0.5.0",
+            "protobuf>=3",
+            "intervaltree",
+            "tenacity>=5",
+        ],
         extras_require={
             # users can include opentracing by having:
             # install_requires=['ddtrace[opentracing]', ...]
             "opentracing": ["opentracing>=2.0.0"],
-            # TODO: remove me when everything is updated to `profiling`
-            "profile": _profiling_deps,
```

Comment:
do we need to keep these as empty list? will `ddtrace[profiling]` still work after removing these?

Reply:
Yes, you can pass any extra it'll just do nothing. :)